src/ReinforcementLearningZoo/src/algorithms/policy_gradient/mpo.jl

```diff
@@ -111,17 +111,17 @@ function update_critic!(p::MPOPolicy, batches)
         s, s′, a, r, t, = send_to_device(device(p.qnetwork1), batch)
         γ, τ = p.γ, p.τ
 
-        a′ = p.actor(p.rng, s′; is_sampling=true, is_return_log_prob=false)
+        a′ = RLCore.estimate_reward(p.actor, p.rng, s′; is_sampling=true, is_return_log_prob=false)
```

Comment:
I'm going to use MPO because I know the algorithm but this is valid for all actor critics. The other issue that I see with estimate reward is that now we use this function to call actor networks too. But this does not estimate a reward nor a return. It outputs an action. Therefore, `estimate_return` is not a better name. Concretely, this function only does a forward pass to the underlying model. So we could simply call it that? `forward`, like in pytorch?

Reply:
Thanks for thinking about this...I was somewhat unsure about the proper naming. Do you want two methods, `estimate_return` and `forward` or should everything be `forward`?

Reply:
The good news is that it's just a find-replace to fix it, but I would like to get the API naming (as close to) right as possible with this PR.

Reply:
I'd love to merge this PR change (https://github.com/JuliaReinforcementLearning/ReinforcementLearning.jl/pull/884) into this PR, what are your thoughts?

Reply:
The easiest I think is to stay with `forward` for now. The distinction between estimate_return and estimate_policy could be useful, be it only for api clarity, but would require a distinction between value approximators and policy approximators. I think that's a bit too much for this already massive PR (119 files changed oof). Interesting for later though, as I'm kind of questioning the usefulness of Approximator in the the first place.